scripts/src/blockchain.ts

```diff
@@ -105,9 +105,9 @@ export class Operations {
 		this.server = server;
 	}
 
-	public async send(operation: xdr.Operation<Operation>, memoText?: string): Promise<Server.TransactionRecord> {
+	public async send(operation: xdr.Operation<Operation>, options: { memo?: string, fee?: number } = {}): Promise<Server.TransactionRecord> {
```

Comment:
i see that this options type: `{ memo?: string, fee?: number }` is used at least 3 times. 
just extract it into a type:
```
type TransactionsOptions = { memo?: string, fee?: number; };
```